src/main/kotlin/com/avail/CallbackSystem.kt

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
The original Java file does not mention @Nullable anywhere, so I believe every "?" in this file should be removed.